src/components/atoms/CardInfoWithBoard/CardInfoWithBoard.tsx

```diff
@@ -0,0 +1,21 @@
+import * as styles from "./CardInfoWithBoard.css";
+
+import { ThumbsUp, ThumbsDown, MessageSquare } from "react-feather";
+
+export const CardInfoWithBoard: React.FC<{
+  board: string;
+  writer: string;
+  time: string;
+}> = ({ board, writer, time }) => {
+  return (
+    <div className={styles.contentInfo}>
+      <div className={styles.infoes}>
+        <div className={styles.board}>{board}</div>
+        <div className={styles.divider}>·</div>
+        <div className={styles.writer}>{writer}</div>
+        <div className={styles.divider}>·</div>
+        <div className={styles.time}>{time}</div>
```

Comment:
style 확인해보니까 모두 동일한 것 같아서 .. 이런 식으로 쓰면 어떨까요? 그리고 `div` 대신 `span` 쓰는 게 더 자연스러운 것 같아요!

```suggestion
        <span className={styles.mainInfo}>{board}</span>
        <span className={styles.subInfo}>·{writer}·{time}</span>
```

Reply:
넵!